study-builder/studies/pancan/i18n/en.conf

```diff
@@ -703,6 +703,7 @@
 "label_firstname": "First Name",
 "label_lastname": "Last Name",
 "hint_diagnosed_cancers": "Please provide diagnosed cancer(s)"
+"hint_duplicated_cancers": "Please do not provide duplicated cancer(s)"
```

Comment:
@MocanaAtBroad what should be a correct hint text (both en/es)?
`"hint_duplicated_cancers": "Please do not provide duplicated cancer(s)"`

Reply:
Please check your entries for duplicates.